### DISCUSSION TITLE
Hospital and Practitioner Information in NDJSON format?

### DISCUSSION THREAD

**Discussion by calidiaz0596** [synthetichealth/synthea#1068] [Q&A]
Hello, 

Excuse me if this is a dumb question, but is there a setting in the configuration file to have the hospital and practitioner information file export as a NDJSON file instead of JSON file? 

I tried manually adding "exporter.hospital.bulk_data = true" to the list to see if this would do anything, but no luck. 

I have also tried just taking the json files and convert them using an online converter but I keep getting the same error:  "Exception Occured: t.map is not a function"

Any help how I could export or convert these 2 files? 

![Screen Shot 2022-05-19 at 10 26 01 AM](https://user-images.githubusercontent.com/79172075/169335734-1fe249df-44a3-4855-b75b-9f65c07e9f3a.png)
![Screen Shot 2022-05-19 at 10 25 49 AM](https://user-images.githubusercontent.com/79172075/169335737-aa17d5a0-67e8-44f7-87b6-2e4365536909.png)

**Comment by jawalonoski** [comment]
No, the hospital and practitioner files are specifically excluded. We did this because this was what most of our users have typically asked for. They POST those two files to a server so those common dependencies appear once, and then bulk load everything else using JSON.

We probably should make a separate parameter to cover those files, but one is not currently available.

**Reply by calidiaz0596** [reply]
Thank you for the response!

**Comment by neerajshri78** [comment]
Hi 

Does synthea currently supports genmerated FHIR data in NDJSON format

**Reply by jawalonoski** [reply]
Yes, but it is off by default.
https://github.com/synthetichealth/synthea/blob/27e32d48da1f2667f91c89964fc575a239cea64b/src/main/resources/synthea.properties#L24
